skbuild/resources/cmake/FindF2py.cmake

```diff
@@ -0,0 +1,65 @@
+# Find the f2py program, which comes as part of NumPy.
```

Comment:
This originally was part of the PyNE package.  Please include the copyright statement, as per the BSD license. Thanks!

Reply:
Done.

Reply:
Thanks so much @xoviat!